sql/core.go

```diff
@@ -695,3 +695,64 @@ type NameableNode interface {
 	Nameable
 	Node
 }
+
+var StatusVariables StatusVariableRegistry
+
+// StatusVariableRegistry is a registry of status variables.
+type StatusVariableRegistry interface {
+	NewSessionMap() map[string]StatusVarValue
+	GetGlobal(name string) (StatusVariable, interface{}, bool)
+	SetGlobal(name string, val interface{}) error
+}
+
+// StatusVariableScope represents the scope of a status variable.
+type StatusVariableScope byte
+
+const (
+	StatusVariableScope_Global StatusVariableScope = iota
+	StatusVariableScope_Session
+	StatusVariableScope_Both
+)
+
+type StatusVariable interface {
```

Comment:
(minor) Even though this interface is super simple, it's still worth taking a couple of minutes to add some doc strings. If it's important enough to be an interface, it's generally worthwhile to add a sentence or two for the interface and methods.

(Alternatively, you could probably get rid of this interface and just expose the struct with this name. I see you matched the new interface for system vars that was added to give Doltgres more flexibility, but I don't think Postgres has the same concept of "status variables" like MySQL does, so it's probably not strictly needed. I don't have a strong opinion either way, and would probably just leave it in like you have it, just to match SystemVariables, but figured I'd mention it.)